From 3805d9909d6d76eba2152f2e9e312b756a9d4b64 Mon Sep 17 00:00:00 2001
From: John Clark <inindev@gmail.com>
Date: Fri, 22 Jan 2016 19:18:40 +0000
Subject: [PATCH 1/6] ramips: HLK-RM04 - Remove power LED config

The power LED on the HLK-RM04 is hard wired to the power bus and is not under GPIO control, remove the bogus config for it.
(Note that GPIO0 is actually connected to the RESET button.)

Signed-off-by: John Clark <inindev@gmail.com>
---
 target/linux/ramips/dts/HLKRM04.dts | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/target/linux/ramips/dts/HLKRM04.dts b/target/linux/ramips/dts/HLKRM04.dts
index f90a9ac..7996f99 100644
--- a/target/linux/ramips/dts/HLKRM04.dts
+++ b/target/linux/ramips/dts/HLKRM04.dts
@@ -94,13 +94,4 @@
 			linux,code = <0x198>;
 		};
 	};
-
-	gpio-leds {
-		compatible = "gpio-leds";
-
-		power {
-			label = "hlk-rm04:red:power";
-			gpios = <&gpio0 0 1>;
-		};
-	};
 };
-- 
2.4.3

